build-test.cmd

```diff
@@ -263,7 +269,7 @@ echo %__MsgPrefix% Restored CoreCLR product from packages
 
 :SkipRestoreProduct
 
-if defined __SkipManaged exit /b 0
+if defined __SkipManaged goto BuildTestHost
 
```

Comment:
Can you follow the convention and `goto SkipManagedBuild` instead of `goto BuildTestHost`?

This makes it easier to add new sections and have the names remain sane.